nextflow_schema.json

```diff
@@ -66,22 +66,40 @@
         "transcript_identification_options": {
             "title": "Transcript Identification Options",
             "type": "object",
-            "description": "",
-            "default": "",
+            "description": "Type of experiment to use for Transcript Identification(NT or TSS)",
             "properties": {
-                "transcript_identification": {
-                    "type": "string",
-                    "default": "homer",
-                    "fa_icon": "fas fa-microscope",
-                    "description": "Method to use for transcript Identification",
-                    "enum": ["homer", "grohmm"]
+                "assay_type": {
+                    "type": "string",
+                    "fa_icon": "fas fa-vial",
+                    "enum": [
+                        "CoPRO",
+                        "GROcap",
+                        "PROcap",
+                        "CAGE",
+                        "NETCAGE",
+                        "RAMPAGE",
+                        "csRNAseq",
+                        "STRIPEseq",
+                        "PROseq",
+                        "GROseq",
+                        "R_5",
+                        "R_3",
+                        "R1_5",
+                        "R1_3",
+                        "R2_5",
+                        "R2_3"
+                    ]
```

Comment:
Do these relate to the FIXME comments above? Does it make sense to have the full list already if not all are supported (or is this for something else?)

Reply:
They're supported in PINTS, but I don't have homer support for example.